docs/fieldnorm/requirements.md

```diff
@@ -0,0 +1,20 @@
+# Requirements in software development
+The complexity and variety of developments in the field of bioinformatics, comes with a need for clear prerequisites before the start of a project. These requirements of software should describe the prerequisites that the software should meet and they should be clear before starting on a project. The requirements may relate to several aspects, including:
+- the performance of the desired software
+- the physical characteristics
```

Comment:
Can you specify "physical characteristics" in terms of software? What do you mean?